.gitignore

```diff
@@ -92,6 +92,7 @@ dd-java-agent.jar
 /.env
 /.env.dev
 /flags.yml
+/temporal/dynamicconfig/development.yaml
```

Comment:
I don't understand why it is in the gitignore.

Reply:
the script is currently set up to copy over all the files we need each time.

it felt cleaner to follow this pattern than commit one thing out of the 5 things we need to spin up the platform

Reply:
yeah we don't want to be checking in platform files here. These are just the bare minimum of files required to run platform

Reply:
👍 I missed that we were in airbyte/airbyte